modules/ocf_mirrors/manifests/timer.pp

```diff
@@ -4,6 +4,7 @@
   $day = '*',
   $month = '*',
   $year = '*',
+  $type = 'sync',
```

Comment:
Maybe make this an Enum type so that it can't be given some other value that doesn't exist?

Not that it really matters though, this is only used in the description of the timer anyway.

Reply:
Yeah I think I'll just leave it as a string for now